config/middleware.py

```diff
@@ -19,3 +21,41 @@ def __call__(self, request):
         # the view is called.
 
         return response
+
+
+class FeedbackLinkMiddleware:
+
+    BASE_FEEDBACK_URL: str = (
+        "https://corexmsnp4n42lf2kht3.qualtrics.com/jfe/form/SV_0lyyYAzfv9bGcyW"
```

Comment:
A part of me is saying "this should be an environment variable", another says it's totally not worth it.

Reply:
I had the exact same thought, and concluded "eh". If we change it in future we can revisit it, at least it's still in exactly one place.